services/player-service/adapter-kafka/src/main/java/com/kjeldsen/player/kafka/events/UserSignedUpEvent.java

```diff
@@ -0,0 +1,18 @@
+package com.kjeldsen.player.kafka.events;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+
+@Getter
+@NoArgsConstructor
+public class UserSignedUpEvent {
+
+    private String id;
+    private Instant occurredAt;
```

Comment:
occurredAt -> ocurrió en .... And it should definitely be part of the common events library. Both Id and date - all of them should have always a value. Maybe eventId and eventDate are good names

Reply:
Also, I added here, because in player code, we have some Event models created in `mongo-adapter` module. Just don't want to change something to avoid break training feature.